documentation/src/docs/asciidoc/release-notes/release-notes-5.5.0-M2.adoc

```diff
@@ -44,6 +44,8 @@ on GitHub.
   the log in order to allow reproducible builds.
 * Methods ordered with `MethodOrderer.Random` now execute using the `SAME_THREAD`
   concurrency mode instead of the `CONCURRENT` mode when no custom seed is provided.
+* Global jupiter configuration `junit.jupiter.displayname.generator.default`
+to override the default `DisplayNameGenerator` for entire test plan.
```

Comment:
Formatting does not adhere to conventions.

Reply:
Applying conventions :)